scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/StaxJobFactory.java

```diff
@@ -167,39 +168,33 @@ public Job createJob(URI filePath, Map<String, String> replacementVariables,
         }
     }
 
+    @Override
+    public Job createJob(InputStream workflowStream) throws JobCreationException {
+        return createJob(workflowStream, null, null);
+    }
+
+    @Override
+    public Job createJob(InputStream workflowStream, Map<String, String> replacementVariables,
+            Map<String, String> replacementGenericInfos) throws JobCreationException {
+        try {
+            return createJobFromInputStream(workflowStream, replacementVariables, replacementGenericInfos);
+        } catch (JobCreationException jce) {
+            throw jce;
```

Comment:
I don't think that those lines are useful:
```java
catch (JobCreationException jce) {
            throw jce;
}
```
Can you remove them ?

Reply:
If I remove this catch clause, all JobCreationException will be caught in the Exception clause and thus will be wrapped into a new instance of JobCreationException (resulting in two nested JobCreationException instances). So I believe there is still value in considering two cases distinctly. What do you think?

Reply:
Seems fair.